COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@eduarrrd Ah so it’s additional terms of contract. Yes, in US you can probably do that.

It could be argued that you’re not competing with Google with this contribution, but without seeing the contract etc. etc. it’s just guesswork. You’d probably need to get a lawyer, if you wanted to try that.

(also, of course, neither this or my post above is legal advice)